lib/haskell/natural4/src/LS/Types.hs

```diff
@@ -739,3 +752,6 @@ enumLabels_ :: ParamText -> [Text.Text]
 enumLabels_ = fmap (Text.replace " " "_") . enumLabels
 
 makeBaseFunctor ''RelationalPredicate
```

Comment:
I may be wrong, but `makeBaseFunctor` is the kind of thing that seems advanced enough that it may be worth leaving a quick link to more info about it. (In particular, I'm worried that this wouldn't be familiar enough to most of the people on the team.) That said, it might also just be the kind of thing one can google by oneself --- I don't have enough experience to have a strong opinion here.

Reply:
I have overlooked this definitiion and I am honestly confused why this even works, `RelationalPredicate` isn't using the `recursion-scheme` style? 0.o
Let me look into it, I will either delete or add some documentation.

Reply:
If Inari hasn't gone on holiday yet, we could try asking her!

Reply:
Found it, added a comment. It makes sense to have it :)